apps/gr_satellites

```diff
@@ -292,6 +297,61 @@ class gr_satellites_top_block(gr.top_block):
     def setup_kiss_input(self):
         self.input = datasources.kiss_file_source(self.options.kiss_in)
 
+    def setup_sigmf_input(self):
+        if self.options.sigmf_in.endswith('sigmf'):
+            print(
+                "Failed to load SigMF archive (archives are not yet supported). "
+                "Consider extracting a recording and pass the corresponding SigMF metadata file.",
+                file=sys.stderr)
+            sys.exit(1)
+        if not self.options.sigmf_in.endswith('sigmf-meta'):
+            print("SIGMF_IN has invalid file extension, expected '.sigmf-meta'", file=sys.stderr)
```

Comment:
Maybe we can print the actual `self.options.sigmf_in` that the user passed instead of the placeholder `SIGMF_IN`. That would make it even more obvious for the user what is wrong with the CLI arguments.